README.md

```diff
@@ -62,6 +64,56 @@ Ready-to-use, copy-paste examples with explanations.
 
 🔹 For more examples check the [full examples](https://constructor-io.github.io/constructorio-ui-autocomplete/?path=/docs/autocomplete-component--docs)
 
+## Customization
+
+By default, importing react components or hooks from this library does not pull any css into your project. To use some starter styles from this library, add an import statement similar to the example import statement below:
+
+```tsx
+import '@constructor-io/constructorio-ui-autocomplete/styles.css';
+```
+
+All the included styles in this library are scoped within the `.cio-autocomplete` css selector, so you can extend existing styles by adding styles to it.
+
+```css
+/* Custom Style Sheet */
+.cio-autocomplete .cio-submit-btn {
+  border-radius: 10px;
+  border: 1px solid red;
+}
+
+.cio-autocomplete .cio-submit-btn:hover {
```

Comment:
I'm thinking we need to keep it short and mabye the above styles are enough as an example. Let me know what you think?